x-pack/test/api_integration/config.ts

```diff
@@ -14,7 +14,6 @@ export async function getApiIntegrationConfig({ readConfigFile }: FtrConfigProvi
   );
 
   return {
-    testFiles: [require.resolve('./apis')],
```

Comment:
Does this file need to be renamed to `config.base.ts`? Seeing that new entry in the buildkite config.

Reply:
I initially renamed it to `config.base.ts`, but it turned to be re-used as basic outside the `x-pack/integration_tests/` path and I decided to keep it as to avoid changes in other config files.
Thank you for closer look, I fixed yml reference in 082f7bd20b5b0cdbe923ef2dd76012531a28b890